lib/v2/screens/explore_screens/vote_screens/proposals/components/proposal_open_card.dart

```diff
@@ -0,0 +1,294 @@
+                                Flexible(
+                                  child: Text('Votes: ' + proposal.total.toString(),
+                                      style: Theme.of(context).textTheme.subtitle3Opacity),
+                                ),
+                              ],
+                            ),
+                          ),
+                          Expanded(
+                            flex: 2,
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.end,
+                              children: [
+                                Padding(
+                                  padding: const EdgeInsets.only(right: 6.0),
+                                  child: CustomPaint(size: const Size(20, 20), painter: VotesDownArrow()),
+                                ),
+                                Flexible(
+                                  child: Text('Against: ' + proposal.againstPercent,
+                                      style: Theme.of(context).textTheme.subtitle3LightGreen6),
+                                ),
+                              ],
+                            ),
+                          ),
+                        ],
+                      ),
+                      const SizedBox(height: 20.0),
+                      Row(
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.only(right: 10.0),
+                            child: CustomPaint(size: const Size(28, 28), painter: VotesTimeIcon()),
+                          ),
+                          Flexible(
+                            child: Text('TODO days left', style: Theme.of(context).textTheme.headline7LowEmphasis),
+                          ),
+                        ],
+                      ),
+                      const SizedBox(height: 20.0),
+                      FlatButtonLong(title: 'View Details and Vote'.i18n, enabled: true, onPressed: () {}),
+                      const SizedBox(height: 16),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class ProposalCategory extends CustomPainter {
+  @override
+  void paint(Canvas canvas, Size size) {
+    Paint paint_0 = Paint()
+      ..color = AppColors.orangeYellow
+      ..style = PaintingStyle.fill
+      ..strokeWidth = 1;
+
+    Path path_0 = Path();
+    path_0.moveTo(0, 0);
+    path_0.lineTo(0, size.height);
+    path_0.lineTo(size.width - 1, size.height);
+    path_0.lineTo(size.width, size.height - 1);
+    path_0.lineTo(size.width - 10, size.height - 11);
+    path_0.lineTo(size.width, 1);
+    path_0.lineTo(size.width - 1, 0);
+    path_0.lineTo(0, 0);
+    path_0.close();
+
+    canvas.drawPath(path_0, paint_0);
+  }
+
+  @override
+  bool shouldRepaint(covariant CustomPainter oldDelegate) => false;
+}
```

Comment:
This time I decided to use canvas instead of svgs. What do you thing? only 💯 % dart code.

Reply:
I like it. Where did u read about this?

Reply:
CustomPainter is a widget that I already knew with the category label has a cut design and figma did not give me the svg because I knew I had to use CustomPainter. For the rest icons.. a further research I found a tool that converts svgs to Dartcode.

Reply:
I liked using it here.
But I do not know if we have it as another option since much of the app is in svgs.

I have not found much clear information about whether it is heavier to flutter load an svg or paint it on canvas.

Reply:
@fbarzin Do you know which has better performance or lower cost? If loading an svg or painting at a low level with the canvas?

Reply:
Flutter doesn’t have the SVG vectorial format as 1st class citizen and that means that SVG is not supported out of the box. For using SVG we’d have to add the flutter_svg package

Reply:
From Flutter offical channel.
![image](https://user-images.githubusercontent.com/42857405/121730182-f3e92900-cab4-11eb-9e8b-d2fc1752fea4.png)